test1.py

```diff
@@ -1,88 +1,188 @@
+    elif exercise.get('category') == 8:
+        arms_exercises_list.append(exercise)
+    elif exercise.get('category') == 14:
+        calves_exercises_list.append(exercise)
+    else:
+        abs_exercises_list.append(exercise)
+
+
+get_workout = login1.get_req('workout')
+for workout in get_workout.get('results'):
+    if workout.get('name') == name_of_workout:
+        id1 = workout.get('id')
+        add_day1 = login1.add_day(id1,"Push1",1)
+        add_day2 = login1.add_day(id1,"Pull1",2)
+        add_day3 = login1.add_day(id1,"Legs1",3)
+
+days = login1.get_req('day',0,200)
+for day in days.get('results'):
+    if day.get('training') == id1:
+        id_day = day.get('id')
+        for i in range(6):
+            number_of_set = 4
+            order = 2
+            req4 = login1.add_exercise(day.get('id'),number_of_set,order)
+
+        list_of_exercises_day = list()
+        if day.get('description') == 'Push1':
+            list_of_exercises_day.append(random.choice(back_exercises_list))
+            list_of_exercises_day.append(random.choice(chest_exercises_list))
+            list_of_exercises_day.append(random.choice(sholders_exercises_list))
+            list_of_exercises_day.append(random.choice(abs_exercises_list))
+            list_of_exercises_day.append(random.choice(chest_exercises_list))
+            list_of_exercises_day.append(random.choice(arms_exercises_list))
+        elif day.get('description') == 'Pull1':
+            list_of_exercises_day.append(random.choice(back_exercises_list))
+            list_of_exercises_day.append(random.choice(back_exercises_list))
+            list_of_exercises_day.append(random.choice(chest_exercises_list))
+            list_of_exercises_day.append(random.choice(abs_exercises_list))
+            list_of_exercises_day.append(random.choice(sholders_exercises_list))
+            list_of_exercises_day.append(random.choice(arms_exercises_list))
+        elif day.get('description') == 'Legs1':
+            list_of_exercises_day.append(random.choice(back_exercises_list))
+            list_of_exercises_day.append(random.choice(legs_exercises_list))
+            list_of_exercises_day.append(random.choice(chest_exercises_list))
+            list_of_exercises_day.append(random.choice(legs_exercises_list))
+            list_of_exercises_day.append(random.choice(calves_exercises_list))
+            list_of_exercises_day.append(random.choice(abs_exercises_list))
+        pprint(list_of_exercises_day)
+
+        exercise_comments = login1.get_req('exercisecomment')
+        comment_limit = exercise_comments.get('count')
+        exercise_images = login1.get_req('exerciseimage')
+        images_limit = exercise_images.get('count')
+        all_exercise_comments = login1.get_req('exercisecomment',offset=0,limit=comment_limit)
+        all_exercise_images = login1.get_req('',offset=0,limit=images_limit)
+        f = open(f"{day.get('description')}_comments.txt","w")
```

Comment:
Read about context managers in Python. They will help you gracefully close this file in case the code breaks before your line 160.